package.json

```diff
@@ -90,7 +90,7 @@
     "write-file-webpack-plugin": "4.5.1"
   },
   "dependencies": {
-    "@modusoperandi/licit-ui-commands": "^0.1.16",
+    "@modusoperandi/licit-ui-commands": "^0.1.13",
```

Comment:
Why go 3 versions backwards? This actually won't work because its a patch update and the carrot will pull all new versions up to the minor changes including patches.

Reply:
Ok, we shall resolve the version conflicts and let you know..